src/Microsoft.TemplateEngine.Edge/TemplateUpdates/NupkgInstallUnitDescriptor.cs

```diff
@@ -36,16 +52,18 @@ public NupkgInstallUnitDescriptor(Guid mountPointId, string packageName, string
             {
                 Dictionary<string, string> detailsInfo = new Dictionary<string, string>()
                 {
-                    { nameof(MountPointId), MountPointId.ToString() },
-                    { nameof(PackageName), PackageName.ToString() },
-                    { nameof(Version), Version }
+                    { nameof(NuGetPackageId), NuGetPackageId },
```

Comment:
Nit: Key could be something more user friendly, though it's not required to be - nor would it necessarily be localized